packages/block-editor/src/components/border-radius-control/style.scss

```diff
@@ -27,31 +27,39 @@
 			}
 
 			.components-range-control__wrapper {
-				margin-right: 10px;
+				margin-right: 12px;
 			}
 		}
 
 		> span {
 			flex: 0 0 auto;
 		}
-	}
 
-	.components-border-radius-control__input-controls-wrapper {
-		display: flex;
-		width: 70%;
-		flex-wrap: wrap;
+		&.has-40px-default-size {
+			.components-range-control {
+				.components-base-control__field {
+					height: 40px;
+					display: flex;
+					align-items: center;
+				}
+			}
 
-		.components-border-radius-control__tooltip-wrapper {
-			width: calc(50% - #{ $grid-unit-10 });
-			margin-bottom: $grid-unit-10;
-			margin-right: $grid-unit-10;
+			.component-border-radius-control__linked-button.has-icon {
+				margin-top: 8px;
+			}
```

Comment:
Thanks, I was just reusing the selectors that were already floating around this component. I don't think it is too hard to refactor away from them if we can allow a custom classname on the UnitControl wrapper to avoid potentially brittle CSS. 

I've done this in https://github.com/WordPress/gutenberg/pull/41860/commits/a97e5624ea677d8d39d668e48a9cb73a1bd26c71 if you're interested or have other ideas.

Reply:
Nice!

It also poses the interesting question on what is the assumption for the `className` prop: should it be the classname of the wrapper component (in `UnitControl`, the outer wrapper), or should it be the classname for the "main" component (in `UnitControl`, the inner input wrapper)  ?

Should this reasoning apply to all components in `@wordpress/components` ?

Reply:
I usually see the `className` apply to the outer wrapper. I think that convention makes sense, since `className` is mostly for (at least somewhat hacky) internal style overrides, and scoping it to the outermost wrapper allows for the most flexibility.

Reply:
Agreed with Lena, and I'd be up for applying that reasoning to `UnitControl` too

But unfortunately `UnitControl` already applies `className` to the inner input wrapper — changing the prop to apply to the outer wrapper may cause some breaking changes unfortunately

Reply:
Let's rename `wrapperClassName` to `__unstableWrapperClassName`, so that this PR can move forward. We will take a look at potentially ironing out this inconsistency in `UnitControl` separately

Reply:
Thank you for applying those changes!

Opened #45094 to track this follow-up task